### ISSUE TITLE
Staging Review finding: Headings that aren't followed by useful content

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#105156]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** MHV
**Product:** Medications
**Feature:** Refill alerts & trackers

## Findings details
**VA.gov Experience Standard - issue:** [User encounters content that meets best practices.](https://design.va.gov/about/experience-standards/#user-encounters-content-that-meets-best-practices)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

Per [design system guidance for the process list component](https://design.va.gov/components/process-list#additional-accessibility-considerations-for-va), make sure each item in the process list has content that follows the heading. Screen reader users often navigate the page skipping heading to heading, and then if they hear a heading that sounds like what they're looking for they'll have their screen reader announce the page content starting from that point. When there's no content that comes after a heading, users might conclude that something is broken or still loading.

<details><summary>Link, screenshot or steps to recreate</summary>

![Refill request status with warning alert and process list. In the process list, only the current step has content after the heading.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/101329-Brian-1.png?raw=true)

</details>

## Recommended action

Each step of the process list needs content to follow the heading. I realize some of the steps don't have obvious content to provide when they're not the active step. The content here can be short and perfunctory when appropriate, but it's important to provide something besides the heading.

I believe this was resolved in the design work shared at your Midpoint Review. Not sure where the regression happened.

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by IdeaTrail** [comment]
This issues is being addressed with Jira ticket [MHV-68045](https://jira.devops.va.gov/browse/MHV-68045).

**Comment by TannerBird98** [comment]
Work is done with above ticket, this can be closed.
